app/components/pages/createPoll/styles.js

```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  CustomFullWidth: {
+    width: '100%',
```

Comment:
i suggested you another datepicker you can add fullwidth=true attribute no need of css